Modules/_sqlite/connection.c

```diff
@@ -1817,6 +1817,116 @@ pysqlite_connection_create_collation_impl(pysqlite_Connection *self,
     Py_RETURN_NONE;
 }
 
+#ifdef PY_SQLITE_HAVE_SERIALIZE
+/*[clinic input]
+_sqlite3.Connection.serialize as serialize
+
+    *
+    name: str = "main"
+        Which database to serialize.
+
+Serialize a database into a byte string. Non-standard.
```

Comment:
What does "non-standard" mean here? I see it on a lot of other docstrings in this file, but SQLite's own documentation doesn't say anything similar.

Reply:
It's a historic artefact; Gerhard marked all non-PEP-249 methods with "non-standard". Let's just remove those in another PR; I know it is confusing to some (most?) people.

Reply:
See GH-31612